README.md

```diff
@@ -35,9 +35,12 @@ Install the golang SDK using go
 go get github.com/harness/ff-golang-server-sdk
 ```
 
-### A Simple Example
-Here is a complete example that will connect to the feature flag service and report the flag value every 10 seconds until the connection is closed.  
-Any time a flag is toggled from the feature flag service you will receive the updated value.
+### Code Sample
+The following is a complete code example that you can use to test the harnessappdemodarkmode Flag you created on the Harness Platform. When you run the code it will:
```

Comment:
Maybe _harnessappdemodarkmode_ should be marked up as code
```suggestion
The following is a complete code example that you can use to test the `harnessappdemodarkmode` Flag you created on the Harness Platform. When you run the code it will:
```